packages/core/migrations/17_deploy_perpetual_creator.js

```diff
@@ -49,4 +51,6 @@ module.exports = async function(deployer, network, accounts) {
   );
 
   await registry.addMember(RegistryRolesEnum.CONTRACT_CREATOR, perpetualCreator.address);
+
+  fs.writeFileSync("../minter/test/uma-contract-address.json", JSON.stringify(addresses, null, 2));
```

Comment:
Basically this sort of becomes a fixture that the tests can use after truffle migrate right? We need to probably put the minter together with the protocol if we need to run this in the CI pipeline. 

cc: @ctverceles

Reply:
before approving, is this also in sync with the latest updates from UMA? @schystz

Reply:
@ernestpascual Yes, I just pulled & merged UMA's master to our master branch yesterday, and branch out from there.

For CI, as discussed earlier, we can have a separate step to checkout the protocol repo, do the setup + start chain + truffle migrate, before running the `test:local` command on minter.